### PR TITLE
feat(conf): [sc 2979] configure filter and limiter rate in cluster mode

### PR DESCRIPTION
**Pull request by mahad-akhter**
- Let the leader initiate the configuration values in KV store
- Update API's to get configuration values from KV store
- Add checks to retrieve configuration values from KV store during initialization if they are available
- Remove in memory configuration values
- Add a generic function to get the values  of provided keys from a key value store

### CODE REVIEW

File: nozl/core/core.go [hunk 1]
```diff
@@ -58,6 +58,34 @@ func (c *core) InitStores(replicationFactor int) {
...
+
+func (c *core) InitConf() {
+	kv, err := eventstream.Eventstream.RetreiveKeyValStore(shared.ConfigKV)
+	if err != nil {
+		shared.Logger.Error(err.Error())
+	}
+
+	_, err = kv.Get(shared.UserTokenRate)
+	if err != nil {
+		kv.Put(shared.UserTokenRate, []byte("1"))
```
Comment (taimoor-mirza) [inline nozl/core/core.go]: @mahad-akhter replace static values with a variable with value `.

File: nozl/core/handler.go [hunk 2]
```diff
@@ -82,8 +91,13 @@ func SetFilterConf(ctx echo.Context) error {
 		})
 	}
 
-	shared.UserTokenRate = limit
-	shared.UserBucketSize = limit
+	kv, err := eventstream.Eventstream.RetreiveKeyValStore(shared.ConfigKV)
+	if err != nil {
+		shared.Logger.Error(err.Error())
```
Comment (taimoor-mirza) [inline nozl/core/handler.go]: In case of error, function should return it and exit instead of continuing.

File: nozl/core/core.go (hunk 1 above)
Comment (mahad-akhter) [reply nozl/core/core.go]: Done

File: nozl/core/handler.go (hunk 2 above)
Comment (mahad-akhter) [reply nozl/core/handler.go]: Done

**Pull request merged**